languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -37502,6 +37502,73 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
+      <rule id="CONFUSION_EST_ET" name="confusion est et" default="temp_off">
```

Comment:
Nice! I can't believe we don't find this error yet.

There is an FP in the rule editor, though:

> Pourquoi elle **et** pas son homonyme géorgienne ?

Reply:
I added the exception and checked the rule editor for more FPs.